### PR TITLE
Bpmn modeller UI changes

### PR DESCRIPTION
**Pull request by devin-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-1586
Issue Type: BUG/ FEATURE

# Changes
UI changes from markups:
[](https://xd.adobe.com/view/04320d3f-50b2-4088-9878-ce1d63fb0942-5161/screen/d9f4560a-add4-4fea-8024-8ae3e8a7f200)

Includes:
- ~~Show non-executable processes in dropdown~~
- Include option to show Internal processes in dropdown (in source code, not ui)
- Dropdown height extends when modeller is shown
- Trash icon set to visible
- Max height set for canvas
- Added "Help" button that redirects to camunda documentation

### CODE REVIEW

**Review by sumesh-aot — APPROVED** [review]

**Review by sreehari-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/apiManager/services/processServices.js [hunk 1]
```diff
@@ -63,10 +64,11 @@ export const getProcessStatusList = (processId, taskId) => {
  *
  * @param  {...any} rest
  */
-export const fetchAllBpmProcesses = (...rest) => {
+export const fetchAllBpmProcesses = (excludeInternal = true, ...rest) => {
   const done = rest.length ? rest[0] : () => {};
   return (dispatch) => {
-    httpGETRequest(API.GET_BPM_PROCESS_LIST + "?latestVersion=true&excludeInternal=true", {}, UserService.getToken(), true)
+    // eslint-disable-next-line max-len
+    httpGETRequest(API.GET_BPM_PROCESS_LIST + `?latestVersion=true&excludeInternal=${excludeInternal}`, {}, UserService.getToken(), true)
```
[the remark this answers is not in the document]
Comment (devin-aot) [reply forms-flow-web/src/apiManager/services/processServices.js:71]: Will do. I didn't see these rules on the master branch, could we copy the contributing.md from the develop branch to the master?

[the remark this answers is not in the document]
Comment (sreehari-aot) [reply forms-flow-web/src/apiManager/services/processServices.js]: We will go for a release this month.

**Review by abhilash-aot — APPROVED** [review]

**Review by sreehari-aot — APPROVED** [review]

File: forms-flow-web/src/apiManager/endpoints/index.js [hunk 2]
```diff
@@ -42,6 +43,9 @@ const API = {
   GET_FORM_COUNT: `${WEB_BASE_URL}/form/<mapper id>/application/count`,
   UNPUBLISH_FORMS: `${WEB_BASE_URL}/form/<mapper id>`,
   DEPLOY_BPM: `${BPM_BASE_URL}/deployment/create`,
+  GET_BPM_DEPLOYMENTS: `${BPM_BASE_URL}/deployment`,
```
Comment (abhilash-aot) [inline forms-flow-web/src/apiManager/endpoints/index.js:46]: @

Comment (abhilash-aot) [reply forms-flow-web/src/apiManager/endpoints/index.js:46]: @SnehaSuresh-aot Check if deployment APIs need to be added to bpm gateway

**Review by abhilash-aot — APPROVED** [review]

**Pull request merged**